osu.Game/Scoring/EFScoreInfo.cs

```diff
@@ -105,7 +105,7 @@ public APIMod[] APIMods
         public string ModsJson
         {
             get => JsonConvert.SerializeObject(APIMods);
-            set => APIMods = JsonConvert.DeserializeObject<APIMod[]>(value);
+            set => APIMods = !string.IsNullOrEmpty(value) ? JsonConvert.DeserializeObject<APIMod[]>(value) : Array.Empty<APIMod>();
```

Comment:
I think it's easier to invert this?

Reply:
How do you mean?

Reply:
`if (!true) { a } else { b }` -> `if (true) { b } else { a }`

Just personal preference. So I'd prefer this to be written `= string.IsNullOrEmpty(value) ? ... : ...`.